viz_scripts/plots.py

```diff
@@ -181,16 +181,16 @@ def overeall_energy_impact(x,y,color,data,plot_title,file_name):
     
     
     
-def energy_impact(x,y,color,plot_title,file_name):
+def energy_impact(x,y,color,plot_title,file_name,xl='Energy_Impact(kWH)'):
```

Comment:
please rename to `impact_plot` or similar, remove the existing `CO2_impact` and replace it by a call to this function.

Reply:
All I did was add a parameter to this one. I didn't actually refactor it. Do you still want to change the name?

Reply:
You also changed the way in which the labels are generated, so it works for any metric, not just energy, right?
So you should change the name, and also replace the implementation of the existing `co2_impact` function with a call to this one.